app/Resources/translations/messages.en.yml

```
@@ -181,4 +181,5 @@ speaker_from_previous_event: "We filled the form with the profile you used on a
 "QR Codes scannés": "Scanned QR Codes"
 "Télécharger en CSV": "Download CSV"
 "Supprimer": "Delete"
-"Lancer l'outil de scan de QR Codes": "Run QR Code Scanner"
\ No newline at end of file
+"Lancer l'outil de scan de QR Codes": "Run QR Code Scanner"
+sponsor.qr_code_scanner.disclaimer: Please confirm with the participant that he or she agrees to you scanning their badge!
```

En version non genrée :
```suggestion
sponsor.qr_code_scanner.disclaimer: Please confirm that the participant agrees for you to scan their badge!
```